app/k8s_utils.py

```diff
@@ -7,7 +7,7 @@ class K8sAPIError(Exception):
     pass
 
 
-def cordon_all_nodes(client, exclude_node_id: str):
+def cordon_all_nodes(client, protect_eviction_disabled, exclude_node_id: str):
```

Comment:
protect_eviction_disabled: bool

Reply:
Changed it to a string and updated name.